doc/content/getting-started/programming-software/st-examples/mounting/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "USB Mounting (Drag-and-Drop)"
+description: ""
+weight: 3
+distributions: null
+---
+
+The ST-LINK probe can work similarly to a USB storage device, where you can simply place data on from your computer. When you place a binary executable on it, it will automatically program it onto the {{% gnse %}}. This technique is suitable for anyone that does not want to install any extra software.
+
+<!--more-->
+
+{{< note >}} Drag-and-Drop functionality is not available for early ST-LINK versions or programmers that run older firmware. If your ST-LINK supports does support this feature, then it will immediately show itself on your computer as a USB storage device after plugging it in. {{</ note >}}
```

Comment:
```suggestion
{{< note >}} Drag-and-Drop functionality is not available for early ST-LINK versions or programmers that run older firmware. If your ST-LINK does support this feature, then it will immediately show itself on your computer as a USB storage device after plugging it in. {{</ note >}}
```